addons/l10n_cn_city/__init__.py

```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
```

Comment:
coding line is not necessary in python 3

Reply:
odoo still tries to be compatible with python 2